safe_cli/operators/safe_operator.py

```diff
@@ -811,6 +874,61 @@ def execute_tx(self, safe_tx_hash: Sequence[bytes]) -> bool:
     def get_permitted_signers(self) -> Set[str]:
         return set(self.safe_cli_info.owners)
 
+    def drain(self, to: str):
+        # Getting all events related with ERC20 transfers
+        events = self.ethereum_client.erc20.get_total_transfer_history(
+            from_block=1, addresses=[self.address]
+        )
+        token_addresses = []
```

Comment:
Why not using a `set` here instead of using a `list` and then converting o a `set`?

Reply:
Make sense

Reply:
Are you using a set then? 😃

Reply:
I'm talking about `token_addresses = []`